runtime/adopted_build.go

```
@@ -0,0 +1,768 @@
+	// 4. A clean detached checkout, and the tree RECOMPUTED from it. The
+	// recomputation is the point: it catches a checkout that does not actually
+	// hold the revision it claims.
+	tree, err := revisionTree(deps, request.RepositoryDir, source)
+	if err != nil {
+		return out, err
+	}
+	checkout, err := os.MkdirTemp("", "zenchron-adopted-build-")
+	if err != nil {
+		return out, err
+	}
+	defer os.RemoveAll(checkout)
```

golangci-lint (2.13.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)